components/AboutAuthor/AboutAuthor.js

```diff
@@ -0,0 +1,42 @@
+import Image from "next/image";
+
+import kid from "../../public/kid.png";
+import styles from "./AboutAuthor.module.css";
+
+// mock data/api
+const fetchMockData = (slug) => ({
+  id: 1234,
+  authorAvatar: kid,
+  authorUsername: "trinh",
+  authorHandle: "@trinhandandy",
+  authorDescription:
+    "When the tangerine moose first heard about the concept of 'orange is the new black,' he couldn't help but feel a little personally offended.",
+  authorLinks: ["x", "y", "z"],
```

Comment:
would these be social links? and we'd eventually display them as icons only?

Reply:
yeah that was the idea. This was the inspiration.

![image](https://user-images.githubusercontent.com/59131637/230548265-1cf9e961-18c2-48e1-a919-b4b917b14682.png)